product_docs/docs/pgd/5/consistency/eager.mdx

```diff
@@ -54,9 +54,9 @@ In case of an origin node failure, the remaining nodes eventually (after at leas
 
 With single-node Postgres, or even with PGD in its default asynchronous
 replication mode, errors at `COMMIT` time are rare. The added synchronization 
-step due to the use of a commit scope using eager
+step due to the use of a commit scope using Eager
```

Comment:
eager is lower case in the commit scope.

Reply:
```suggestion
step due to the use of a commit scope using eager
```
Should eager be tagged? Is it an option?

Reply:
```suggestion
step due to the use of a commit scope using `eager`
```